CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,206 @@
+This document is a declaration of software quality for the [Eclipse Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds) (hereafter simply "Cyclone DDS") external dependency imported by the `rmw_cyclonedds` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# Cyclone DDS Quality Declaration
+
+Cyclone DDS meets all the requirements of **Quality Level 2** category insofar as they are relevant to the `rmw_cyclonedds` package and ROS 2's use of `rmw_cyclonedds` and `cyclonedds`. Cyclone DDS meets all of the requirements of Quality Level 3 category for uses outside of the scope of ROS 2. The former follows from the implication that the features needed for a ROS 2 tier-1 DDS-based middleware must necessarily be supported, and those features being covered by tests.
+
+The requirements for the various quality level categories are defined by [REP-2004](https://www.ros.org/reps/rep-2004.html). The rationale, notes and caveats for this claim are provided by the remainder of this document.
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+Cyclone DDS version numbers are organised as MAJOR.MINOR.PATCH where all three components are non-negative decimal numbers.  Version number policy follows the following rules:
```

Comment:
For consistency with the rest of the document, use a single space after a period.

```suggestion
Cyclone DDS version numbers are organised as MAJOR.MINOR.PATCH where all three components are non-negative decimal numbers. Version number policy follows the following rules:
```